### ISSUE TITLE
Wallabag 2.6.2 CSS not loading in admin panel

### ISSUE THREAD

**Issue by jameswill** [wallabag/wallabag#6755]
### Environment

* **Version**: 2.6.2
* **Installation**: Downloading the package
* **PHP version**: 7.4
* **Database**: 5.7.42
* **Parameters**:

<details>
  <summary>My <code>app/config/parameters.yml</code> is:</summary>

```
# This file is auto-generated during the composer install
parameters:
    database_driver: pdo_mysql
    database_host: 127.0.0.1
    database_port: null
    database_name: wallabag262
    database_user: root
    database_password: mysql
    database_path: null
    database_table_prefix: wallabag_
    database_socket: null
    database_charset: utf8mb4
    domain_name: 'http://example.com/wallabag262/web'
    server_name: 'Your wallabag instance'
    mailer_dsn: 'smtp://127.0.0.1'
    locale: en
    secret: CHANGE_ME_TO_SOMETHING_SECRET_AND_RANDOM
    twofactor_sender: no-reply@wallabag.org
    fosuser_registration: true
    fosuser_confirmation: true
    fos_oauth_server_access_token_lifetime: 3600
    fos_oauth_server_refresh_token_lifetime: 1209600
    from_email: no-reply@wallabag.org
    rss_limit: 50
    rabbitmq_host: localhost
    rabbitmq_port: 5672
    rabbitmq_user: guest
    rabbitmq_password: guest
    rabbitmq_prefetch_count: 10
    redis_scheme: tcp
    redis_host: localhost
    redis_port: 6379
    redis_path: null
    redis_password: null
    sentry_dsn: null
```
</details>

### What steps will reproduce the bug?

I have downloaded the package using wget **https://www.wallabag.org/latest-v2-package** and installed it on my test server with above details. However after completing the installation when I visit the admin panel, it's CSS seems to be broken throwing following error in console.

**_Uncaught ReferenceError: fos is not defined
    <anonymous> http://example.com/wallabag262/web/js/routing?callback=fos.Router.setData:1_**

![wallabag](https://github.com/wallabag/wallabag/assets/6368036/f0044518-c16a-442b-8da1-8bb54f3d4886)

**Comment by j0k3r** [comment]
Could you check the network instead?

**Comment by jameswill** [comment]
![image](https://github.com/wallabag/wallabag/assets/6368036/e045c89f-da9d-4c82-b920-4d920872125d)

**Comment by jameswill** [comment]
![image](https://github.com/wallabag/wallabag/assets/6368036/a1de0af4-d41c-4130-8dc3-3c87acd46b61)

**Comment by jameswill** [comment]
I have set domain_name in parameters.yml and hide it out in my previous response because of the security.

Also tried clearing /var/logs but the issue persists.

**Comment by j0k3r** [comment]
Oh sorry, I mean clear the folder `/var/cache` 🤦🏼

**Comment by jameswill** [comment]
Hi,

Thank you. The above issue seems to be resolved after clearing cache. However after logging into admin panel few links stills brakes down. (Top right corner)

![image](https://github.com/wallabag/wallabag/assets/6368036/80e9e734-c2de-4748-a228-cf45c84e9c06)

**Comment by j0k3r** [comment]
Well, the `domain_name` seems to be different than the one used to access wallabag. It must be the same domain otherwise fonts won't load (as you can see).

**Comment by jameswill** [comment]
Can you please point out where are you finding different `domain_name` ? As I can see I have set **domain_name: 'https://example.com/wallabag262/web'** in parameters.yml and same domain I am trying to access via UI.

**Comment by j0k3r** [comment]
That URL doesn't seems to be the same as the one in `domain_name`.
<img width="418" alt="image" src="https://github.com/wallabag/wallabag/assets/62333/f36fb86c-6998-49fb-bfca-ae2956ef86b4">

**Comment by jameswill** [comment]
I have used the same url in parameters.yml (Double checked) just hided here with "example.com" while posting as it is a public platform.

**Comment by jameswill** [comment]
Hi,

I checked and found that that was happening because misconstruing of the symlinks in /web/bundles.

I got it correct and installation working fine now. Thank you for your help.

However I am facing following issue in upgrading the wallabag 2.5.4 to 2.6.2 Should I continue here or open a new issue ?

Fatal error: Declaration of Wallabag\UserBundle\Controller\RegistrationController::registerAction(Symfony\Component\HttpFoundation\Request $request) must be compatible with FOS\UserBundle\Controller\RegistrationController::registerAction(Symfony\Component\HttpFoundation\Request $request): Symfony\Component\HttpFoundation\Response in [[PATH]]/src/Wallabag/UserBundle/Controller/RegistrationController.php on line 10
